app/src/test/java/com/team7/cmput301/android/theirisproject/RecordTest.java

```diff
@@ -45,6 +40,7 @@ public void testRecord(){
 
     @Test
     public void testAddPhoto() {
+
         Record record = getTestRecord();
 
         record.addPhoto(null);
```

Comment:
Might be better to actually add an actual RecordPhoto inside (you can just instantiate a RecordPhoto using some random text for the blob) and ensure that the properties of the Record you added into the list are the same as the one you get out.